src/commands/export/fasta.rs

```diff
@@ -0,0 +1,59 @@
+use crate::commands::cli_context::CliContext;
+use crate::commands::get_db_for_command;
+use crate::config::get_operation_connection;
+use crate::exports::fasta::export_fasta;
+use crate::get_connection;
+use crate::models::metadata;
+use crate::models::operations::setup_db;
+use clap::Args;
+use rusqlite::Connection;
+use std::path::PathBuf;
+
+/// Export a FASTA file
+#[derive(Debug, Args)]
+pub struct Command {
+    /// FASTA file path
+    #[clap(index = 1)]
+    pub path: String,
+    /// The name of the collection for exporting
+    #[arg(short, long)]
+    name: Option<String>,
+    /// The name of the sample for exporting
+    #[arg(short, long)]
+    sample: Option<String>,
+}
+
+fn get_default_collection(conn: &Connection) -> String {
+    let mut stmt = conn
+        .prepare("select collection_name from defaults where id = 1")
+        .unwrap();
+    stmt.query_row((), |row| row.get(0))
+        .unwrap_or("default".to_string())
+}
+
+pub fn execute(cli_context: &CliContext, cmd: Command) {
+    println!("GFA export called");
+    let operation_conn = get_operation_connection(None);
```

Comment:
I have plans for more refactoring here, like to get all the starting boilerplate into a separate method, but this PR got big enough already